packages/expo-clipboard/android/src/test/java/expo/modules/clipboard/ClipboardModuleTest.kt

```diff
@@ -24,77 +20,79 @@ import org.robolectric.annotation.Implementation
 import org.robolectric.annotation.Implements
 import org.robolectric.shadows.ShadowContextImpl
 
+private interface ClipboardModuleTestInterface {
+  @Throws(CodedException::class)
+  fun getStringAsync(): String
+
+  @Throws(CodedException::class)
+  fun setStringAsync(content: String): Boolean
+}
```

Comment:
Is it possible, in the future, to generate this interface from the module definition? 🤔

Reply:
Yes and no. Technically speaking it is, but I'm not sure if that's worth doing. To achieve that we have to add two more packages - one for annotation processor and another one for moving some part of the new API to the pure java library.  The second step is needed to invoke the module definition function to get all the types. Instead of this, we could parse the source code of the module definition, but that idea sounds too complicated for my liking.